docs/src/main/docs/config/04_property-mapping.adoc

```diff
@@ -261,13 +266,29 @@ uses Java reflection to search for those ways to perform the mapping.
 Your application facilitates this implicit mapping either by adding to the 
 POJO class or by providing a builder class for it.
 
+This feature is available in Object mapping module, and is added through java service
+load mechanism. This is no longer part of core Config module, as it depends on reflection
```

Comment:
Did you mean the "Java ServiceLoader" class? If yes, please use that specific term.